SagaFlow.UI/package.json

```diff
@@ -6,15 +6,19 @@
   "scripts": {
     "dev": "vite",
     "build": "vite build",
+    "build:web-components": "vite build --config ./vite.webComponents.config.ts",
     "preview": "vite preview",
     "check": "svelte-check --tsconfig ./tsconfig.json"
   },
   "devDependencies": {
     "@sveltejs/vite-plugin-svelte": "^2.0.4",
     "@tsconfig/svelte": "^4.0.1",
     "@types/node": "^20.2.6",
+    "sass": "^1.77.4",
+    "scss": "^0.2.4",
```

Comment:
We shouldn't need scss. what are we using it for?

Reply:
fixed